tests/testthat/test_civis_ml_utils.R

```diff
@@ -106,3 +110,27 @@ test_that("get_train_template_id reverts to last id if others not available", {
   expect_equal(id, ans)
 })
 
+test_that("get_feature_importance returns correct feature importance matrix", {
+  ans_function <- function(m){
+        variable_order <- order(m$metrics$model$parameters$feature_importances)
+        variable_names <- m$metrics$model$parameters$relvars[rev(variable_order)]
+        importance <- m$metrics$model$parameters$feature_importances[rev(variable_order)]
+
+        feature_importance_df <- data.frame('importance' = importance)
+        rownames(feature_importance_df) <- variable_names
+        feature_importance_df
+        }
```

Comment:
I think you should precompute all the feature importance data and save it in `tests/testthat/data`.  

```r
importances <- lapply(models, get_feature_importance)
saveRDS("tests/testthat/data/feature_importances.rds")
```

And then you can call them here with

```r
feature_importances <- readRDS("data/feature_importances.rds")
for (i in 1:length(feat_imp_mods)) expect_equal(feat_imp_mods[[i]], feature_importances[[i]])
```